web/concrete/src/Asset/JavascriptAsset.php

```diff
@@ -1,80 +1,95 @@
 <?php
 namespace Concrete\Core\Asset;
+
 use HtmlObject\Element;
 use Config;
 
-class JavascriptAsset extends Asset {
+class JavascriptAsset extends Asset
+{
+
+    protected $assetSupportsMinification = true;
+    protected $assetSupportsCombination = true;
 
-	protected $assetSupportsMinification = true;
-	protected $assetSupportsCombination = true;
+    protected $scriptType = 'text/javascript';
```

Comment:
to be able to subclass this class and change this in the output without having to rewrite the class output method.

(Well saying that I should have refactored the code to create a ScriptAsset class and make the JavasriptAsset derives from it … but wouln't that make the code too much bloated?)